README.md

```diff
@@ -70,6 +76,8 @@ julia> MeCab.libmecab
 ```
 
 ## Credits
+September 2018, modified for Julia 1.0 by snthot
+
```

Comment:
After merging it, I will add "contributors" section on README, so could you remove it?

Reply:
well, removed.
I'm new, please forgive me.